src/crowdstrike/foundry/function/_internal/__init__.py

```diff
@@ -0,0 +1,21 @@
+config = None
```

Comment:
is ther any way we can remove the need for globals and make these fields on an object? Trying to get away from globals, as they're typically a bad practice.